packages/server/src/service/org/passport.ts

```diff
@@ -244,10 +245,39 @@ export const registerPassportEndpoints = (app: Express.Application) => {
       const { oid } = req.params
       const uid = getUid(req)
       const stateInfos = await firestoreService.fetchRegistrations(uid, oid) || []
+
```

Comment:
Let's put this on a new endpoint that get's called by a corn job.  We just fetch up to 500 applications and update information.

Can we sort by `.alloyStatus.timestamp` (oldest first) but never after 24 hours ago and then update those?  Then we just have to do 500 every hour and we'll eventually update all the data.

Reply:
Thanks! A draft version for the cronjob is in the works at #188, had to do a bit different than the logic described here since from what I understand some registration status are not final (thus we need to keep a queue for tracking voters that can still be updated)

Reply:
Let’s move this logic into another endpoint so that we’re just introducing a new endpoint